_build/stac.hbs

```diff
@@ -40,6 +40,7 @@
       "href": "{{{find WMTS 'href' 'href'}}}",
       "rel": "wmts",
       "wmts:layer": "{{{find WMTS 'layer_id' 'layer_id'}}}"
+      "wmts:dimensions": {{{find WMTS 'dimensions' 'dimensions'}}}

```

Comment:
I added the `,` at the end of the line above so it's also present in the generated json file (fields in a json object have to be separated with `,`).

I wrapped the `{{{find WMTS 'dimensions' 'dimensions'}}}` with `""` so the value in the generated json file is in `""` (without that, the value was not in `""` in the generated json and because the value contains `:`, json treats it as another field)

Reply:
Makes both sense, thanks! I removed the `""` originally because I toughed they will be copied from the yaml file but see now that it's not the case.